### ISSUE TITLE
[Simplified Collect][Tags] Implement bulk actions for tags

### ISSUE THREAD

**Issue by luacmartins** [Expensify/App#38635]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:**
**Reproducible in staging?:**
**Reproducible in production?:**
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:**
**Slack conversation:**

## Action Performed:
Break down in numbered steps

## Expected Result:
Describe what you think should've happened

## Actual Result:
Describe what actually happened

## Workaround:
Can the user still use Expensify without this being fixed? Have you informed them of the workaround?

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Native
- [ ] Android: mWeb Chrome
- [ ] iOS: Native
- [ ] iOS: mWeb Safari
- [ ] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

Add any screenshot/video evidence

</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by allroundexperts** [comment]
Translation verification required for the following:

**English**

```
enableTags: 'Enable tags',
disableTag: 'Disable tag',
disableTags: 'Disable tags',
deleteTags: 'Delete tags',
deleteTagsConfirmation: 'Are you sure that you want to delete these tags?',
```

**Spanish**

```
  enableTags: 'Habilitar etiquetas',
  disableTag: 'Desactivar etiqueta',
  disableTags: 'Desactivar etiquetas',
  deleteTags: 'Eliminar etiquetas',
  deleteTagsConfirmation: '¿Estás seguro de que quieres eliminar estas etiquetas?',
```

**Similar translations for categories**

```
deleteCategoryPrompt: '¿Estás seguro de que quieres eliminar esta categoría?',
disableCategories: 'Desactivar categorías',
disableCategory: 'Desactivar categoría',
enableCategories: 'Activar categorías',
enableCategory: 'Activar categoría',
deleteFailureMessage: 'Se ha producido un error al intentar eliminar la categoría. Por favor, inténtalo más tarde.',
categoryName: 'Nombre de la categoría',
```

**Screenshots**
![Screenshot 2024-03-20 at 1 31 55 AM](https://github.com/Expensify/App/assets/30054992/dcee8eb5-eee2-4f30-865d-c73b18a5acc5)

![Screenshot 2024-03-20 at 1 32 03 AM](https://github.com/Expensify/App/assets/30054992/cee450d0-6d36-427a-bdbf-e5b0be3bea53)

![Screenshot 2024-03-20 at 1 32 09 AM](https://github.com/Expensify/App/assets/30054992/0b8a2335-9c75-4303-95a8-9182bccb27cb)

![Screenshot 2024-03-20 at 1 32 25 AM](https://github.com/Expensify/App/assets/30054992/cb993dcc-ebf6-4d3a-8a85-3ca9b68e3712)

**Comment by mountiny** [comment]
Hello @shmaxey can you please help us confirm the copy from marketing perspective. Thank you

**Comment by shmaxey** [comment]
On the [English] confirmation prompts, we  should say "Are you sure ~that~ you want to delete these tags?" and "Are you sure ~that~ you want to delete this tag?" 

Otherwise, looks good!

**Comment by iammudassirali** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.
The problem we aim to address in this issue is the absence of bulk actions functionality for tags in Simplified Collect. Currently, users are unable to perform actions like editing or deleting multiple tags simultaneously, leading to inefficiencies and increased manual effort.

### What is the root cause of that problem?
The root cause of this problem lies in the lack of a feature that allows users to select and apply actions to multiple tags at once. 

### What changes do you think we should make in order to solve the problem?
To solve this issue, we should implement bulk actions functionality, enabling users to manage tags more efficiently and streamline their workflow. By adding options to edit, delete, or apply changes to multiple tags simultaneously, we can significantly enhance the user experience and improve productivity.

### What alternative solutions did you explore? (Optional)
Alternative solutions explored include manual tag management and implementing individual actions for each tag, but these approaches are time-consuming and inefficient. Implementing bulk actions provides a more user-friendly and scalable solution to address this issue effectively.

I look forward to discussing this proposal further and collaborating on its implementation.

Best regards,
Mudassir Ali

**Comment by mountiny** [comment]
$500 for @ishpaul777 and for @allroundexperts

**Comment by slafortune** [comment]
@mountiny @allroundexperts I believe no regression tests are needed, can you confirm?

**Comment by mountiny** [comment]
Yeah I think these will be handled in a separate issue for Simplified collect test cases

**Comment by slafortune** [comment]
@ishpaul777 Offer sent [here](https://www.upwork.com/nx/wm/offer/101714482)
@allroundexperts to be paid via NewDot - $500

**Comment by slafortune** [comment]
Moving this to weekly

**Comment by slafortune** [comment]
@allroundexperts to be paid via NewDot - $500

**Comment by JmillsExpensify** [comment]
$500 approved for @allroundexperts
